src/iuse.cpp

```diff
@@ -5628,8 +5628,12 @@ void iuse::play_music( player * const p, const tripoint &source, int const volum
     if( int(calendar::turn) % 50 == 0 ) {
         // Every 5 minutes, describe the music
         auto const music = get_music_description( *p );
-        if( !music.sound.empty() ) {
+        // mp3 or stereo playing on the same square
+        if( !music.sound.empty() && p->pos() == source ) {
             sound = string_format( _("You listen to %s"), music.sound.c_str() );
+        } else if ( !music.sound.empty() ) {
+            // music from other squares, leave only music description
+            sound = string_format( _("%s"), music.sound.c_str() );
```

Comment:
Translating the string "%s" is a bit pointless, the translators have no context information.
Calling `string_format` like this is similar pointless. It will just return the second argument without any changes.

Just write `sound = music.sound;`